### ISSUE TITLE
Username highlights on the dark theme look bad due to brownish colour

### ISSUE THREAD

**Issue by dkasak** [vector-im/element-web#21806]
Username highlights on the dark theme look bad while unselected due to the brownish background colour which stands out from the rest of the theme:

![unselected](https://i.imgur.com/Df5u0o9.png)

It's a bit better when selected since it then changes to a grey shade:

![selected](https://i.imgur.com/Ttc502i.png)

I think this is just dark theme bug stemming from the colour not being handpicked but brought over from the light theme.

I originally mentioned this at https://github.com/vector-im/element-web/issues/8554#issuecomment-524572148_, which got closed.

**Comment by SimonBrandner** [comment]
I believe that the reddish-brownish colour is intentional - it's supposed to catch your attention, the light theme has something similar

**Comment by dkasak** [comment]
The red is not problematic, but the brownish colour just looks completely out of place in combination with the rest of the theme. We never use brownish tones anywhere else and it's (IMO) a particularly ugly shade.

In the light theme, a yellowish shade is used which works much better.

**Comment by t3chguy** [comment]
That looks like its a translucent yellow, which mixed with the bg looks brownish

**Comment by dkasak** [comment]
Yes, which made me think this is because the colour was just brought over from the light theme without any adjustments to make it look good on the dark theme.
